### PR TITLE
adding fabric

### PR DESCRIPTION
**Pull request by robsonpeixoto**

### CODE REVIEW

File: Library/Formula/fabric.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+require "formula"
+
```
Comment (bfontaine) [inline Library/Formula/fabric.rb]: The build fails because `require "formula"` is now unnecessary.

Comment (robsonpeixoto) [inline Library/Formula/fabric.rb]: Thanks @bfontaine. Just a feature request, add this analysis on brew audit.

Comment (robsonpeixoto) [inline Library/Formula/fabric.rb]: Done in 59aed90

Comment (bfontaine) [inline Library/Formula/fabric.rb]: @robsonpeixoto I’m not a maintainer, and this is in `brew audit` but you have to add the `--strict` option ;)

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! For future reference the preferred commit message format for new formulae is `fabric 1.10.1 (new formula)`.

**Comment by robsonpeixoto** [comment]
Thanks

**Pull request closed without merging**